verify_hound_checks.rb

```diff
@@ -0,0 +1,13 @@
+# Dummy class to violate syntax guidelines
+class VerifyHoundChecks
+  def array_join
+    # bad
+    str = [:a, :b, :c] * ','
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.